lib/src/Uncertainty/StatTests/FittingTest.cxx

```diff
@@ -259,6 +259,15 @@ Distribution FittingTest::BestModelBIC(const Sample  & sample,
 Distribution FittingTest::BestModelKolmogorov(const Sample & sample,
     const DistributionFactoryCollection & factoryCollection,
     TestResult & bestResult)
+{
+  LOGWARN("FittingTest::BestModelKolmogorov(const Sample & sample, const DistributionFactoryCollection & factoryCollection, TestResult & bestResult) is deprecated, use FittingTest::BestModelLilliefors(const Sample & sample, const DistributionFactoryCollection & factoryCollection, TestResult & bestResult)");
```

Comment:
shorter version:
"FittingTest::BestModelKolmogorov(Sample, DistributionFactoryCollection, TestResult) is deprecated, use BestModelLilliefors"